android/app/src/main/java/org/mozilla/magnet/db/History.java

```diff
@@ -0,0 +1,192 @@
+package org.mozilla.magnet.db;
+
+import android.content.ContentValues;
+import android.content.Context;
+import android.database.Cursor;
+import android.database.sqlite.SQLiteDatabase;
+import android.database.sqlite.SQLiteOpenHelper;
+import android.util.Log;
+
+import org.mozilla.magnet.db.Schema.HistoryEntry;
+
+import java.util.Date;
+import java.util.concurrent.TimeUnit;
+
+import static android.provider.BaseColumns._ID;
+import static org.mozilla.magnet.db.Schema.HistoryEntry.COLUMN_NAME_TIME_LAST_SEEN;
+import static org.mozilla.magnet.db.Schema.HistoryEntry.COLUMN_NAME_URL;
+import static org.mozilla.magnet.db.Schema.HistoryEntry.COLUMN_NAME_TIME_FIRST_SEEN;
+
+public class History extends SQLiteOpenHelper {
```

Comment:
Yeah, I would call it HistorySQLite, so if we realise that we need to replace SQLite by ... XYZ, we just replace the HistoryXYZ object that implements the logic.

But we keep the high level semantics of History, the want that we want to be used by other classes.